<?xml version="1.0" encoding="UTF-8"?>
<!--
QAccountant - personal accounting software
Copyright (C) 2025  Cole Blakley

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
-->
<ui version="4.0">
 <class>TransactionsView</class>
 <widget class="QFrame" name="TransactionsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>387</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="toolbar">
     <item>
      <widget class="QToolButton" name="new_transaction">
       <property name="toolTip">
        <string>New Transaction</string>
       </property>
       <property name="text">
        <string>New Transaction</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListAdd"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="delete_transaction">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Delete Transaction</string>
       </property>
       <property name="text">
        <string>Delete Transaction</string>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::ListRemove"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="submit_changes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Submit Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="revert_changes">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Revert Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="transactions_view">
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
